blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/Export.java

```diff
@@ -98,4 +119,73 @@ private static void writeOne(TreePruner pruner, DataWriter dw, Object item) thro
     }
 
     private Export() {};
+
+    public static class BlueOceanExportInterceptor extends ExportInterceptor{
+        @Override
+        public Object getValue(Property property, Object model) throws IOException {
+            if(model instanceof Action){
+                try {
+                    Object value = property.getValue(model);
+
+                    //If {@link CustomExportedBean} then return just the String value
+                    if(value instanceof CustomExportedBean){
+                        return ((CustomExportedBean) value).toExportedObject();
+                    }
+
+                    //Check for ExportedBean annotation
+                    if(checkForExportedBean(value) && (value.getClass().getAnnotation(ExportedBean.class) == null)){
+                        throw new NotExportableException(value.getClass(), model.getClass(), property.name);
+                    }
```

Comment:
@vivek Shouldn't you return `value` here so that you don't convert it twice on line 144?